src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
+  padding: 0;
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.18);
+  height: 60px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.box {
+  text-align: center;
+  position: relative;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  padding: 20px 0;
+  height: 60px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  margin: 0;
+}
+
+.container {
+  position: relative;
+  margin-right: 40px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+
+  margin-left: 50px;
+}
+
+nav ul li {
+  position: relative;
+  margin: 0 10px;
+}
+
+nav ul li:first-child {
+  margin-left: 0;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 5px;
+  background-color: rgba(0, 150, 200, 1);
+  border-radius: 5px;
+  color: rgba(0, 150, 200, 1);
```

Comment:
The 'color' property in the '.is-active::after' selector is redundant since the pseudo-element does not contain any text. You can remove this property to clean up the code.